skimage/_shared/tests/test_utils.py

```diff
@@ -300,3 +300,27 @@ def test_decorator_warnings():
 
     assert record[0].lineno == expected_lineno
     assert record[0].filename == __file__
+
+
+@deprecate_func(
```

Comment:
```suggestion
@deprecated_func(
```
I'm not sure about my suggestion: I think that *formally* it helps with consistency (`deprecated_func` replaced `deprecated`, plus the dummy function below is named `_deprecated_func`), but maybe you intended that and you are absolutely logical (the decorator *deprecates* while the dummy function stands for a *deprecated* function).

Reply:
It seems okay to leave as `deprecate_func`, but no strong opinion either way

Reply:
@grlee77 ok, thanks.
@lagru feel free to ignore this suggestion.